content/SCALE/SCALETutorials/Apps/CommunityApps/MinIOApp/MinIOClustering.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Setting Up MinIO Clustering"
+description: "Provides configuration instructions using the MinIO Offical Charts application widget. It includes instructions on setting up a distributed cluster configuration."
+weight: 20
+aliases:
+ - /scale/scaleuireference/apps/minioclustersscale/
+ - /scale/scaletutorials/apps/minioclustersscale/minioclustering/
+tags:
+- scaleminio
+- scaleapps
+- scaleclustering
+---
+
+{{< toc >}}
+
+
+{{< hint info >}}
+This article applies to the public release of the S3 **MinIO** charts application in the TRUENAS catalog.
+{{< /hint >}}
+
+On TrueNAS SCALE 23.102-ALPHA.2 and later, users can create a MinIO S3 distributed instance to scale out and handle individual node failures. 
+A node is a single TrueNAS storage system in a cluster.
+
+The examples below use four TrueNAS systems to create a distributed cluster.
+For more information on MinIO distributed setups, refer to the [MinIO documentation](https://docs.min.io/docs/distributed-minio-quickstart-guide.html).
+
+## First Steps
+
+{{< include file="/_includes/MinIODatasetRequirements.md" type="page" >}}
+
+For a distributed configuration, repeat this on all system nodes in advance. 
+
+Take note of the system (node) IP addresses or host names and have them ready for configuration. Also, have your S3 user name and password ready for later.
+
+## Configuring MinIO
+
+Configure the MinIO application using the full version **Minio charts** widget. 
+Go to **Apps**, click **Discover Apps** then 
+
+We recommend using the **Install** option on the **MinIO** application widget. 
+
+If your system has sharing (SMB, NFS, iSCSI) configured, disable the share service before adding and configuring a new MinIO deployment. 
+After completing the installation and starting MinIO, enable the share service.
+
+If the dataset for the MinIO share has the same path as the MinIO application, disable host path validation before starting MinIO. 
+To use host path validation, set up a new dataset for the application with a completely different path. For example, for the share */pool/shares/minio* and for the application */pool/apps/minio*.
+
+### Configuring MinIO Using Install
+
+Begin on the first node (system) in your cluster. 
+
+{{< include file="/_includes/MinIOInstallAppNameWorkloadConfigSteps.md" type="page" >}}
+
+Next, enter the **MinIO Configuration** settings.
+
+{{< include file="/_includes/MinIOEnableDistributedModeInfo.md" type="page" >}}
+
+{{< include file="/_includes/MinIOInstallArgAndEnvironVarSteps.md" type="page" >}}
+
+For a distributed cluster, ensure the values are identical between nodes and fill the **Minio image environment** values with proper random credentials.
```

Comment:
What do you mean by "proper random credentials"? The image environment value in the screenshot doesn't look random to me.

Reply:
it means they can enter any random characters they want for user name or password, but I'll adjust this text to remove confusion.